src/ansys/fluent/core/session_solver.py

```diff
@@ -363,7 +363,10 @@ def __getattr__(self, attr):
                     f"'{attr}' is deprecated. Use 'settings.{attr}' instead.",
                     DeprecatedSettingWarning,
                 )
-        return getattr(self._settings_api_root, attr)
+        try:
+            return getattr(self._settings_api_root, attr)
+        except AttributeError:
+            return object.__getattribute__(self, attr)
```

Comment:
```suggestion
        # Try forwarding attribute access to the settings API root.
        # If that fails with AttributeError, fall back to normal attribute lookup.
        # Using object.__getattribute__ triggers the standard AttributeError with default messaging.
        try:
            return getattr(self._settings_api_root, attr)
        except AttributeError:
            # Let standard attribute access raise the appropriate error
            return object.__getattribute__(self, attr)
```

Reply:
@prmukherj 
Better version of the if statement: `if not attr.startswith("_") and attr in dir(self._settings_api_root):`

Reply:
Done, Thank you.

Reply:
@prmukherj Just checking that you saw the final comment, above.

Reply:
@seanpearsonuk, yes, done.